blazor/multicolumn-combobox/custom-value.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Adding Custom value to the Blazor MultiColumn ComboBox Component | Syncfusion
+description: Checkout and learn here all about adding custom value in Syncfusion Blazor MultiColumn ComboBox component and much more.
+platform: Blazor
+control: MultiColumn ComboBox
+documentation: ug
+---
+
+# Adding Custom Value to Blazor MultiColumn ComboBox Component
+
+You can include custom values in the MultiColumn ComboBox component. If the entered character(s) are not found in the list, a button will appear in the popup menu. Clicking this button will add the custom character(s) to the existing list as a new item. By default, the [AllowCustom]() property is set to `true`.
```

Comment:
**If the entered character(s) are not found in the list, a button will appear in the popup menu** - How is this handled in the code ?